runner-manager/cf-driver/prepare.sh

```diff
@@ -182,24 +194,29 @@ start_service() {
         "$container_id"
         '-m' "$WORKER_MEMORY"
         '--docker-image' "$image_name"
-        '--health-check-type' 'process'
         '--no-route'
     )
 
-    if [ -n "$job_vars" ] || [ -n "$service_vars" ]; then
-        declare -A vars=()
+    SVCMANIFEST=$(mktemp /tmp/gitlab-runner-svc-manifest.XXXXXXXXXX)
+    TMPFILES+=("$SVCMANIFEST")
+    chmod 600 "$SVCMANIFEST"
 
-        SVCMANIFEST=$(mktemp /tmp/gitlab-runner-svc-manifest.XXXXXXXXXX)
-        TMPFILES+=("$SVCMANIFEST")
-        chmod 600 "$SVCMANIFEST"
+    {
+        echo "---"
+        echo "applications:"
+        echo "- name: $container_id"
+        echo "  env:"
+    } >>"$SVCMANIFEST"
 
-        {
-            echo "---"
-            echo "applications:"
-            echo "- name: $container_id"
-            echo "  env:"
-        } >>"$SVCMANIFEST"
-    fi
+    declare -A vars=()
+
+    vars[no_proxy]='localhost,apps.internal'
+    vars[SSL_CERT_FILE]='/etc/ssl/certs/ca-certificates.crt'
+
+    # Add any WSR_SERVICE_x variables populated by start_services()
```

Comment:
[suggestion (nb)] same as above

```suggestion
    # Add any WSR_* variables populated by start_services()
```